### DISCUSSION TITLE
이모티콘 path질문이요!

### DISCUSSION THREAD

**Discussion by ghost** [storycraft/node-kakao#523] [Q&A]
v3은 어태치먼트에 있던데 v4는 따로없는건가요?

**Comment by dn1t** [comment]
있습니다. chat.attachment를 EmoticonAttachment로 캐스팅하면 잘 있어요.

![Screenshot_20210809-095623_GitHub.jpg](https://user-images.githubusercontent.com/45326612/128651307-dc0a26d5-18ed-4442-ae76-70e34047f7eb.jpg)
